site/public/js/calendar.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Gets the previous month of a given month
+ * @param month : int the current month (1 as January and 12 as December)
+ * @param year : int the current year
+ * @returns {number[]} : array<int> {previous_month, year_of_previous_month}
+ */
+// eslint-disable-next-line no-unused-vars
+function prevMonth(month, year) {
+    month = month - 1;
+    if (month <= 0) {
+        month = 12 + month;
+        year = year - 1;
+    }
+    return [month, year];
+}
+
+/**
+ * Gets the next month of a given month
+ * @param month : int the current month (1 as January and 12 as December)
+ * @param year : int the current year
+ * @returns {number[]} : array<int> {next_month, year_of_next_month}
+ */
+// eslint-disable-next-line no-unused-vars
+function nextMonth(month, year) {
+    month = month + 1;
+    if (month > 12) {
+        month = month - 12;
+        year = year + 1;
+    }
+    return [month, year];
+}
+
+/**
+ * Creates a HTML table cell that contains a date.
+ *
+ * @param year : int the year of the date
+ * @param month : int the month of the date (1 as January and 12 as December)
+ * @param day : int the date of the date (1 - 31)
+ * @param curr_view_month : int the current month that the calendar is viewing
+ * @returns {string} the HTML string containing the cell
+ */
+function displayDayCell(year, month, day, curr_view_month) {
+    const cell_date_str = `${year.toString()}-${month.toString().padStart(2, '0')}-${day.toString().padStart(2, '0')}`;
+    let content = `<td class="cal-day-cell" id=${cell_date_str}>
+    <div>`;
+    // Title of the day cell
+    content += '<div>';
+    if (month === curr_view_month) {
+        // eslint-disable-next-line no-undef
+        if (day === curr_day && month === curr_month && year === curr_year) {
+            content += `<span class="cal-curr-month-date cal-day-title cal-today-title">${day}</span>`;
+        }
+        else {
+            content += `<span class="cal-curr-month-date cal-day-title">${day}</span>`;
+        }
+    }
+    else {
+        if (month > 12) {
```

Comment:
why would month ever be greater than 12? Lets assume the caller of this function will pass a month representation of int from 0-11